sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -8,6 +8,8 @@ Azure Cognitive Services Form Recognizer is a cloud service that uses machine le
   - Business Cards - Recognize and extract common fields from business cards, using a pre-trained business cards model.
   - Invoices - Recognize and extract common fields from invoices, using a pre-trained invoice model.
 
+Please note, this package is a replacement for [Microsoft.CognitiveServices.FormRecognizer][old_package].
+
```

Comment:
Wasn't 100% sure if it was worth adding this line since the old package hasn't even gone GA. I don't think it will hurt that much, though.

Reply:
igk. I'm torn with it. I looked at other libraries and there are different approaches. Some mention it by being specific on the service version that the library supports, some ignore old libraries, some have a separate Readme for that.
I will say, it might be better not to include it here to avoid making a user confused about one library or other.
If, for whatever reason, a user is looking for information on the track1 library they will realize that it is not even supported by the service and when looking at service docs they will find us.

Now, if we get feedback that users are confused and by adding this we will help them, then we can add it

Reply:
> it might be better not to include it here to avoid making a user confused about one library or other.

Good point. Let's follow your suggestion, then. It's documentation, so as you said, so it's easy to change it later if needed.

Reply:
sounds good